packages/optimistic-oracle-v2/scripts/build-manifest.sh

```diff
@@ -1,15 +1,31 @@
 #!/bin/bash
 
 NETWORK=$1
-
 FILE=$NETWORK'.json'
-
 DATA=manifest/data/$FILE
 
-echo 'Generating manifest from data file: '$DATA
-cat $DATA
+echo "Generating manifest from data file: $DATA"
+
+# If RENAME_CHAIN_ID is set, update the network field in the JSON data on the fly
+if [[ -n "$RENAME_CHAIN_ID" ]]; then
```

Comment:
allows to replace the chain id name for cases where they do not match between providers, eg. with blast, TheGraph id is `blast-mainnet` but Goldsky is `blast`